lib/dotcom_web/plugs/secure_headers.ex

```diff
@@ -0,0 +1,63 @@
+defmodule DotcomWeb.Plugs.SecureHeaders do
+  @moduledoc """
+  Configures secure browser headers for Dotcom requests, including specifying
+  content security policy directives at funtime.
+  """
+
+  import Phoenix.Controller, only: [put_secure_browser_headers: 2]
+
+  @base_csp_directives [
+    default: "default-src 'none'",
+    img:
+      "img-src 'self' cdn.mbta.com px.ads.linkedin.com www.linkedin.com www.facebook.com *.google.com *.googleapis.com *.gstatic.com *.s3.amazonaws.com data: i.ytimg.com www.googletagmanager.com *.arcgis.com",
+    style: "style-src 'self' 'unsafe-inline' www.gstatic.com cdn.jsdelivr.net",
+    script:
+      "script-src 'self' 'unsafe-eval' 'unsafe-inline' insitez.blob.core.windows.net snap.licdn.com connect.facebook.net www.instagram.com www.google-analytics.com *.google.com www.gstatic.com www.googletagmanager.com *.googleapis.com data.mbta.com *.arcgis.com",
+    font: "font-src 'self'",
+    connect:
+      "connect-src 'self' *.googleapis.com analytics.google.com www.google-analytics.com www.google.com px.ads.linkedin.com stats.g.doubleclick.net *.arcgis.com *.s3.amazonaws.com",
+    frame:
+      "frame-src 'self' data.mbta.com www.youtube.com www.google.com cdn.knightlab.com livestream.com www.instagram.com *.arcgis.com",
+    worker: "worker-src blob: ;"
```

Comment:
Just a suggestion, but since we're cleaning this up, is there a better way to make this manageable? Thinking something like...
```
@base_img_directives ~w[
  img-src
  'self'
  ...
]
```

We can put all of the different types into these all in alphabetical order. Then:

```
@base_csp_directives [
  img: @base_image_directives,
  ...
]
```

Reply:
Would be nice to alphabetize too since these lists are long and have a lot of overlap...multiple references to google.

Reply:
There's lots of overlap between the directives and somehow lots of differences too, i'll give it a crack!

Reply:
didn't do much to account for overlaps since they're so inconsistent, but just alphabetizing and formatting with newlines gave this a big readability win!